create-projects/create-projects.sh

```diff
@@ -1,88 +1,119 @@
 #!/usr/bin/env bash
-set -ex
+set -e
+
+PROJECT_ID=""
+PROJECT_ADMINS=""
+PROJECT_GROUPS=""
+# Role 'admin' is needed to clone an entire project including role bindings
+# for env autoclonding in the Jenkins shared library.
+JENKINS_ROLE="admin"
+
+function usage {
+  printf "usage: %s [options]\n" "$0"
+  printf "\t-h|--help\tPrints the usage\n"
+  printf "\t-v|--verbose\tVerbose output\n"
+  printf "\t-p|--project\tProject ID\n"
+  printf "\t--admins\tAdmins of the projects\n"
+  printf "\t--groups\tGroups with permissions (e.g. 'USERGROUP=foo,ADMINGROUP=bar,READONLYGROUP=baz')\n"
```

Comment:
@clemensutschig Could you confirm whether this is the correct format? I read through the code and this is what I understood from it. Therefore, also the tests assert a value with this format now.